src/devices/legrand.js

```diff
@@ -447,6 +447,30 @@ module.exports = [
         toZigbee: [],
         exposes: [e.action(['press_once', 'press_twice'])],
     },
+    {
+        zigbeeModel: ['GreenPower_3'],
+        fingerprint: [{modelID: 'GreenPower_3', ieeeAddr: /^0x00000000005.....$/}],
+        model: '600087L',
+        vendor: 'Legrand',
+        description: 'Wireless and batteryless blind control switch',
+        fromZigbee: [{
+            cluster: 'greenPower',
+            type: ['commandNotification'],
+            convert: (model, msg, publish, options, meta) => {
```

Comment:
Please move this converter to an `fzLocal` in the top of this file (https://github.com/Koenkk/zigbee-herdsman-converters/blob/3080896a20a743b6bd79287e9fc7f7418e2b3046/src/devices/legrand.js#L29)

Reply:
Ok

Reply:
Done